gtfs-validator/main/src/test/java/org/mobilitydata/gtfsvalidator/validator/AmbiguousStopStationTransfersValidatorTest.java

```diff
@@ -0,0 +1,223 @@
+          .setCsvRowNumber(12)
+          .setStopId("stop2")
+          .setLocationType(GtfsLocationType.STOP.getNumber())
+          .setParentStation("stationB")
+          .build();
+  private final GtfsStop stop3_in_stationC =
+      new GtfsStop.Builder()
+          .setCsvRowNumber(13)
+          .setStopId("stop3")
+          .setLocationType(GtfsLocationType.STOP.getNumber())
+          .setParentStation("stationC")
+          .build();
+  private final GtfsStop boarding_area1_in_stationA =
+      new GtfsStop.Builder()
+          .setCsvRowNumber(14)
+          .setStopId("boarding_area1")
+          .setLocationType(GtfsLocationType.BOARDING_AREA.getNumber())
+          .setParentStation("stationA")
+          .build();
+  private final GtfsStop entrance2_in_stationB =
+      new GtfsStop.Builder()
+          .setCsvRowNumber(15)
+          .setStopId("entrance2")
+          .setLocationType(GtfsLocationType.ENTRANCE.getNumber())
+          .setParentStation("stationB")
+          .build();
+
+  // For transferTable:
+  private final GtfsTransfer transfer_from_stop1_in_stationA_to_stationB =
+      new GtfsTransfer.Builder()
+          .setCsvRowNumber(1)
+          .setFromStopId("stop1")
+          .setToStopId("stationB")
+          .build();
+  private final GtfsTransfer transfer_from_stationA_to_stop2_in_stationB =
+      new GtfsTransfer.Builder()
+          .setCsvRowNumber(2)
+          .setFromStopId("stationA")
+          .setToStopId("stop2")
+          .build();
+  private final GtfsTransfer transfer_from_stop2_in_stationB_to_stationA =
+      new GtfsTransfer.Builder()
+          .setCsvRowNumber(3)
+          .setFromStopId("stop2")
+          .setToStopId("stationA")
+          .build();
+  private final GtfsTransfer transfer_from_stop1_in_stationA_to_stationC =
+      new GtfsTransfer.Builder()
+          .setCsvRowNumber(4)
+          .setFromStopId("stop1")
+          .setToStopId("stationC")
+          .build();
+  private final GtfsTransfer transfer_from_stop3_in_stationC_to_stationB =
+      new GtfsTransfer.Builder()
+          .setCsvRowNumber(5)
+          .setFromStopId("stop3")
+          .setToStopId("stationB")
+          .build();
+  private final GtfsTransfer transfer_from_boarding_area1_in_stationA_to_stationB =
+      new GtfsTransfer.Builder()
+          .setCsvRowNumber(6)
+          .setFromStopId("boarding_area1")
+          .setToStopId("stationB")
+          .build();
+  private final GtfsTransfer transfer_from_stationA_to_entrance2_in_stationB =
+      new GtfsTransfer.Builder()
+          .setCsvRowNumber(7)
+          .setFromStopId("stationA")
+          .setToStopId("entrance2")
+          .build();
+
+  @Test
+  public void sameStationSameDirectionTransfersShouldGenerateNotice() {
+    final NoticeContainer noticeContainer = new NoticeContainer();
+    List<GtfsStop> stops =
+        new ArrayList<>(Arrays.asList(stationA, stationB, stop1_in_stationA, stop2_in_stationB));
```

Comment:
As mentioned, please prefer Arrays.asList or (possibly better) use https://guava.dev/releases/snapshot/api/docs/com/google/common/collect/ImmutableList.html#of-E-

Reply:
Thank you for the advice! I have changed to use ImmutableList now.